### DISCUSSION TITLE
HGTV

### DISCUSSION THREAD

**Discussion by archi821** [iptv-org/community#957] [Q&A]
### Channel ID

HGTVAsia.us

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
The channel has been added to our blocklist due to the claims of the copyright holder: [#1831](https://github.com/iptv-org/iptv/issues/1831)
